application/src/main/resources/application.properties

```diff
@@ -32,4 +32,6 @@ logging.file.name=app.log
 # temp folder example
 #logging.file=${java.io.tmpdir}/app.log
 logging.pattern.file=%d %p %c{1.} [%t] %m%n
-logging.pattern.console=%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n
\ No newline at end of file
+logging.pattern.console=%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n
+keystore.password=test
```

Comment:
Move it to test-application.properties in test resource directory or move it to separate test files in test resource directory